EZEE-2864: schedule drafts and reschedule scheduled content in the dashboard (#48)
ezsystems/user-documentation: docs/publishing/advanced_publishing_options.md

    
    The calendar widget toolbar displays events you select and gives you access to actions assigned to them.
    The available actions appear in the upper-right corner of the toolbar when you select an event.

>     You can select multiple events of the same type and perform bulk actions on them, e.g. [reschedule or cancel publication](#rescheduling-or-cancelling-with-calendar-toolbar).

Reviewer: ```suggestion
    You can select multiple events of the same type and perform bulk actions on them, e.g. [reschedule or cancel publication](#rescheduling-or-cancelling-publications).
```

    
    ![Calendar widget toolbar](img/selection_action_bar.png "Calendar widget toolbar")
    
    ## Rescheduling or cancelling publications
    
    In case of publishable Content items (e.g. articles), you can change or cancel their planned publication using the **Reschedule** or **Unschedule** buttons.

>     These buttons are accessible in the [Dashboard](#rescheduling-or-cancelling-in-dashboard) and the [Calendar widget](#rescheduling-or-cancelling-with-calendar-toolbar).

Reviewer: those topics are just few lines below, do we need links to them?

Author: I may have overdone it. ;)

    
    |Button|Description|
    |------|-----------|
    |![Reschedule button](img/selection_action_bar_reschedule.png)|Reschedule all selected events.|
    |![Discard button](img/selection_action_bar_discard.png)|Cancel the future publication of selected events.|
    
    #### Rescheduling or cancelling with Dashboard
    
    To reschedule or cancel events in the Dashboard, perform the following actions:
    
    1. Open the Dashboard by clicking the eZ logo in the left-upper corner.
    1. In the **Me** panel, view all your scheduled Content items by clicking **My scheduled**.
    1. From **My Publications scheduled**, select all Content items to have their publication time rescheduled or cancelled.

>     1. Using the buttons in the upper-right corner, perform one of the following actions:

Reviewer: ```suggestion
    1. Using the buttons in the upper-right corner, perform one or both of the following actions:
```

>     1. From **My Publications scheduled**, select all Content items to have its publication time rescheduled or cancelled.

Reviewer: ```suggestion
    1. From **My Publications scheduled**, select all Content items to have their publication time rescheduled or cancelled.
```

        - To change the publication time, click the **Reschedule** button.
          In the **Reschedule publication** modal window, select the new date and click **Confirm reschedule**.
        - To cancel publication, click the **Unschedule** button.

>           In the modal window, confirm the cancellation by clicking **Discard scheduling**.

Reviewer: ```suggestion
          In the modal window, confirm the cancellation by selecting **Discard scheduling**.
```

>     ![Reschedule or cancel in Dashboard](img/reschedule_cancel_dashboard.png "Rescheduling or cancelling in the Dashboard")

Reviewer: The buttons are specified in the table, there is no need for a cursor in the screenshot.

    
    #### Rescheduling or cancelling with Calendar toolbar
    

>     In the [Calendar widget](#calendar-widget), select all events to have its publication time rescheduled or cancelled.

Reviewer: ```suggestion
    In the [Calendar widget](#calendar-widget), select all events to have their publication time rescheduled or cancelled.
```
